### DISCUSSION TITLE
Error messages when trying to install Quire (macOS)

### DISCUSSION THREAD

**Discussion by leschram** [thegetty/quire#144] [Get Help] [answered]
Hi all,
I'm getting the following message when I try to update Quire. Any help and advice are most welcome!! Thank you.

leslieschramer@Leslies-MacBook-Pro-2 ~ % xcode-select --install
xcode-select: error: command line tools are already installed, use "Software Update" to install updates
leslieschramer@Leslies-MacBook-Pro-2 ~ % npm install --global @thegetty/quire-cli
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
/usr/local/bin/quire -> /usr/local/lib/node_modules/@thegetty/quire-cli/bin/index.js

> fsevents@1.2.13 install /usr/local/lib/node_modules/@thegetty/quire-cli/node_modules/watchpack-chokidar2/node_modules/fsevents
> node install.js

No receipt for 'com.apple.pkg.CLTools_Executables' found at '/'.

No receipt for 'com.apple.pkg.DeveloperToolsCLILeo' found at '/'.

No receipt for 'com.apple.pkg.DeveloperToolsCLI' found at '/'.

gyp: No Xcode or CLT version detected!
gyp ERR! configure error 
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/configure.js:351:16)
gyp ERR! stack     at ChildProcess.emit (events.js:375:28)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:277:12)
gyp ERR! System Darwin 19.6.0
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /usr/local/lib/node_modules/@thegetty/quire-cli/node_modules/watchpack-chokidar2/node_modules/fsevents
gyp ERR! node -v v14.17.1
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok 

> core-js@3.15.2 postinstall /usr/local/lib/node_modules/@thegetty/quire-cli/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
> https://opencollective.com/core-js 
> https://www.patreon.com/zloirock 

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)

> hugo-bin@0.70.2 postinstall /usr/local/lib/node_modules/@thegetty/quire-cli/node_modules/hugo-bin
> rimraf vendor && node lib/install

✔  success   Hugo binary installed successfully!
+ @thegetty/quire-cli@0.19.2
added 596 packages from 294 contributors in 33.438s
leslieschramer@Leslies-MacBook-Pro-2 ~ %

**Comment by geealbers** [comment, marked as answer]
Hi @leschram, thanks for posting and for providing the shell output. There's a line in there that we've seen before:

```
gyp: No Xcode or CLT version detected!
```

If you're running macOS Catalina, check out this post and try the solutions there: https://medium.com/flawless-app-stories/gyp-no-xcode-or-clt-version-detected-macos-catalina-anansewaa-38b536389e8d. Once you've gotten xcode successfully reset/reinstalled, try the Quire install again and let us know how it goes.

**Reply by leschram** [reply]
That worked! I'm back in business -- thank you, @geealbers !

**Reply by leschram** [reply]
One--hopefully--final issue. Quire installed and I began a new project, but when I try to preview on http://localhost:1313, I only see the HTML version, not the true preview. I can make changes in the yml files and they are reflected on the localhost, but only in text--not rendered.

**Reply by geealbers** [reply]
Hmm. Sounds like it’s not loading the stylesheets. Can you try running `quire preview --verbose` and see if you see any errors in the Terminal output? You can post the results here. 

ps. I’m away from my desk this week so replies might be delayed
